receiver/awscontainerinsightreceiver/internal/stores/utils_test.go

```diff
@@ -98,6 +98,36 @@ func TestUtils_addKubernetesInfo(t *testing.T) {
 	assert.Equal(t, expectedKubeBlob, kubernetesBlob)
 }
 
+func TestUtils_addKubernetesInfoRetainContainerNameTag(t *testing.T) {
+	fields := map[string]interface{}{ci.MetricName(ci.TypePod, ci.CPUTotal): float64(1)}
+	tags := map[string]string{
+		ci.ContainerNamekey: "testContainer",
+		ci.K8sPodNameKey:    "testPod",
+		ci.PodIDKey:         "123",
+		ci.K8sNamespace:     "testNamespace",
+		ci.TypeService:      "testService",
+		ci.NodeNameKey:      "testNode",
+		ci.Timestamp:        strconv.FormatInt(time.Now().UnixNano(), 10),
+	}
+
+	metric := &mockCIMetric{
+		tags:   tags,
+		fields: fields,
+	}
+
+	kubernetesBlob := map[string]interface{}{}
+	AddKubernetesInfo(metric, kubernetesBlob, true)
```

Comment:
Maybe I should define a var and pass that in here instead of `true` to make the test more readable instead of just passing in the bool directly?

Reply:
Maybe have a for loop test that will test both true and false values

Reply:
IMO I think having a for loop that just tests true vs false is excessive.